src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -3,6 +3,12 @@
 import static java.lang.Boolean.TRUE;
 import static java.util.Objects.nonNull;
 import static java.util.Objects.requireNonNull;
+import static org.folio.circulation.domain.LoanAction.CHECKED_OUT;
+import static org.folio.circulation.domain.LoanAction.CHECKED_IN;
+import static org.folio.circulation.domain.LoanAction.DECLARED_LOST;
+import static org.folio.circulation.domain.LoanAction.RENEWED;
+import static org.folio.circulation.domain.LoanAction.RENEWED_THROUGH_OVERRIDE;
+import static org.folio.circulation.domain.representations.LoanProperties.*;
```

Comment:
Why is there a * import as well as specific imports below it.

I believe our provisional import standard does not use * imports at all.

Reply:
Done.